packages/autoloader/composer.json

```diff
@@ -10,6 +10,9 @@
 		"phpunit/phpunit": "^5.7 || ^6.5 || ^7.5"
 	},
 	"autoload": {
+		"files": [
+			"src/function-enqueue-packages.php"
```

Comment:
I don’t think this file needs to be listed here. Files in this list will be required by the [file_loader() method](https://github.com/Automattic/jetpack/blob/master/packages/autoloader/src/autoload.php#L121).  However, we’re going to require this file in the autoload_packages.php file.

Reply:
Removed that in 12360fa86, thank you!